components/Dialogs/src/MDCDialogPresentationController.m

```diff
@@ -244,6 +244,14 @@ - (void)dismissalTransitionWillBegin {
   }
 }
 
+- (CGAffineTransform)dialogTransform {
+  return self.trackingView.transform;
```

Comment:
what if there is no trackingView? or is it always going to be there?

Reply:
`trackingView` is internal to the class and is created on init, so I think it's safe to assume that it will always be present.